db/migrate/20200324094424_change_date_to_timestamp.rb

```diff
@@ -0,0 +1,182 @@
+      change_column :fields_definitions, :updated_at, :datetime
+      change_column :cms_sections, :created_at, :datetime
+      change_column :cms_sections, :updated_at, :datetime
+      change_column :cms_templates, :created_at, :datetime
+      change_column :cms_templates, :updated_at, :datetime
+      change_column :users, :created_at, :datetime
+      change_column :users, :updated_at, :datetime
+      change_column :users, :remember_token_expires_at, :datetime
+      change_column :users, :activated_at, :datetime
+      change_column :users, :last_login_at, :datetime
+      change_column :users, :lost_password_token_generated_at, :datetime
+      change_column :cinstances, :created_at, :datetime
+      change_column :cinstances, :updated_at, :datetime
+      change_column :cinstances, :paid_until, :datetime
+      change_column :cinstances, :trial_period_expires_at, :datetime
+      change_column :cinstances, :variable_cost_paid_until, :datetime
+      change_column :cinstances, :first_traffic_at, :datetime
+      change_column :cinstances, :first_daily_traffic_at, :datetime
+      change_column :cinstances, :accepted_at, :datetime
+      change_column :referrer_filters, :created_at, :datetime
+      change_column :referrer_filters, :updated_at, :datetime
+      change_column :application_keys, :created_at, :datetime
+      change_column :application_keys, :updated_at, :datetime
+      change_column :posts, :created_at, :datetime
+      change_column :posts, :updated_at, :datetime
+      change_column :topics, :created_at, :datetime
+      change_column :topics, :updated_at, :datetime
+      change_column :topics, :last_updated_at, :datetime
+      change_column :billing_strategies, :created_at, :datetime
+      change_column :billing_strategies, :updated_at, :datetime
+      change_column :plans, :created_at, :datetime
+      change_column :plans, :updated_at, :datetime
+      change_column :access_tokens, :created_at, :datetime
+      change_column :access_tokens, :updated_at, :datetime
+      change_column :api_docs_services, :created_at, :datetime
+      change_column :api_docs_services, :updated_at, :datetime
+      change_column :authentication_providers, :created_at, :datetime
+      change_column :authentication_providers, :updated_at, :datetime
+      change_column :cms_groups, :created_at, :datetime
+      change_column :cms_groups, :updated_at, :datetime
+      change_column :cms_files, :attachment_updated_at, :datetime
+      change_column :cms_files, :created_at, :datetime
+      change_column :cms_files, :updated_at, :datetime
+      change_column :alerts, :timestamp, :datetime
+      change_column :line_items, :created_at, :datetime
+      change_column :line_items, :updated_at, :datetime
+      change_column :line_items, :finished_at, :datetime
+      change_column :line_items, :started_at, :datetime, System::Database.postgres? ? {using: 'current_date + started_at'} : {}
```

Comment:
I think this is incorrect ... I will try to check

Reply:
So in Postgres `time` does not convey dates ... that is why we add `current_date` to be able to transform it to `timestamp` (datetime)

https://stackoverflow.com/questions/24930570/postgresql-convert-to-timestamp

Looks OK then

Reply:
At then end, we should not change them